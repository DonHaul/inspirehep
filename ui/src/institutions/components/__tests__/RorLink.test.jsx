import React from 'react';
import { render } from '@testing-library/react';
import RorLink from '../RorLink';

describe('RorLink', () => {
  it('renders', () => {
    const ror = 'https://ror123.org';
    const { asFragment } = render(<RorLink ror={ror} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
