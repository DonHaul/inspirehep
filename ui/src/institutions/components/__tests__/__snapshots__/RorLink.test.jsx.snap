// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RorLink renders 1`] = `
<DocumentFragment>
  <span>
    ROR Record: 
    <a
      href="https://ror123.org"
      target="_blank"
    >
      https://ror123.org
    </a>
  </span>
</DocumentFragment>
`;
